docs/admin-guide/opa.md

```diff
@@ -1,15 +1,60 @@
 `karina.yml`
+#OPA Gatekeeper
 
 ```yaml
-
+gatekeeper:
+  version: v3.3.0
+  constraints: /path/to/constraints/folder
+  templates: /path/to/templates/folder
+  whitelistNamespaces:
+    - unpolicied-namespace
```

Comment:
I think move these out to another section titled, `Adding new constraints`, `Excluding namespaces` with examples of the contents . e.g.